Dockerfile

```diff
@@ -8,15 +8,15 @@ ARG GO_BUILD_ARGS
 COPY . /app
 WORKDIR /app
 
-RUN go get github.com/google/go-licenses && go-licenses save ./... --save_path third_party_licenses --force
-RUN go get -d ./...
+# move previously cached go modules to gopath
+RUN mkdir -p ${GOPATH}/pkg && mv ./mod ${GOPATH}/pkg
```

Comment:
this fails when running `make deploy-local-easy` for me, with the message that `./mod` doesn't exist